### PR TITLE
fix RUN-1074 - allow embedding LOCAL driver binary

### PR DESCRIPTION
**Pull request by Domiii**
* enable `build.js` to also embed a LOCAL driver binary
  * -> https://linear.app/replay/issue/RUN-1074/allow-buildjs-to-embed-local-instead-of-downloaded-driver
* snuck a change to `uploadChromiumBuildToCloudDevelopment.sh` in, to report timings on all individual steps, so we can better understand the exact extent of the bottlenecks standing in the way of fast cloud dev (at the risk that it might not get accepted)

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -1,6 +1,16 @@
+// Copyright 2021 Record Replay Inc.
```
Comment (bhackett1024) [inline build.js:1]: This copyright should have the same BSD license notice that other chromium files and files we've written have.

File: copyLinuxBuild.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+// Copyright 2021 Record Replay Inc.
```
Comment (bhackett1024) [inline copyLinuxBuild.js:1]: ditto

File: uploadChromiumBuildToCloudDevelopment.sh [hunk 3]
```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# Copyright 2021 Record Replay Inc.
```
Comment (bhackett1024) [inline uploadChromiumBuildToCloudDevelopment.sh:2]: ditto

**Review by bhackett1024 — APPROVED** [review]

File: build.js (hunk 1 above)
Comment (Domiii) [reply build.js]: Woops! I was following the wrong project guidelines.
Fixed now.

**Pull request merged**